### PR TITLE
use utils in knative/pkg

### PR DESCRIPTION
**Pull request by runzexia**
Signed-off-by: runzexia <root@junwuhui.cn>

Fixes knative/eventing#2884

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: pkg/reconciler/controller/natsschannel_test.go [hunk 1]
```diff
@@ -19,10 +19,9 @@ package controller
 import (
 	"context"
 	"fmt"
```
Comment (mattmoor) [inline pkg/reconciler/controller/natsschannel_test.go:21]: Format Go code:
```suggestion
	"fmt"
	"testing"

```

File: pkg/reconciler/controller/natsschannel_test.go [hunk 2]
```diff
@@ -19,10 +19,9 @@ package controller
...
+	"knative.dev/pkg/network"
 	"testing"
```
Comment (mattmoor) [inline pkg/reconciler/controller/natsschannel_test.go:23]: Format Go code:
```suggestion

```

File: pkg/reconciler/controller/resources/service.go [hunk 3]
```diff
@@ -18,11 +18,11 @@ package resources
 
 import (
 	"fmt"
```
Comment (mattmoor) [inline pkg/reconciler/controller/resources/service.go:20]: Format Go code:
```suggestion
	"fmt"

```

File: pkg/util/util.go [hunk 4]
```diff
@@ -2,9 +2,8 @@ package util
 
 import (
 	"fmt"
```
Comment (mattmoor) [inline pkg/util/util.go:4]: Format Go code:
```suggestion
	"fmt"
	"os"

```

File: pkg/util/util.go [hunk 5]
```diff
@@ -2,9 +2,8 @@ package util
...
+	"knative.dev/pkg/network"
 	"os"
```
Comment (mattmoor) [inline pkg/util/util.go:6]: Format Go code:
```suggestion

```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

**Review by n3wscott — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**